### PR TITLE
{devel}[GCC/4.9.2] make v4.1

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
This was used in my short demo of `eb --new-pr`, but it's still useful to merge it in.

@wpoely86: please review

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5805ff61a7302e17bff7 for a full test report.

File: easybuild/easyconfigs/m/make/make-4.1-GCC-4.9.2.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+##
+easyblock = 'ConfigureMake'
+
+name = 'make'
+version = '4.1'
+
+homepage = 'http://www.gnu.org/software/make/make.html'
+description = "GNU version of make utility"
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/make/make-4.1-GCC-4.9.2.eb]: `GCCcore`

File: easybuild/easyconfigs/m/make/make-4.1-GCC-4.9.2.eb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+sources = ['make-%(version)s.tar.bz2']
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/make/make-4.1-GCC-4.9.2.eb]: `SOURCE_TAR_BZ2`?

File: easybuild/easyconfigs/m/make/make-4.1-GCC-4.9.2.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/m/make/make-4.1-GCC-4.9.2.eb]: This is 4.9.2, not 4.9.3.

Also, if someone needs this with `GCCcore`, they can add it.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5f3b38637233832ed67c for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**